src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.FreeBSD.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <NativeBinary Include="$(BinDir)libcoreclr.so" />
+    <NativeBinary Include="$(BinDir)libcoreclr.so" LinkerUsage="runtime" />
     <NativeBinary Condition="'$(_PlatformDoesNotSupportEventTrace)' != 'true'" Include="$(BinDir)libcoreclrtraceptprovider.so" />
     <NativeBinary Include="$(BinDir)libdbgshim.so" />
```

Comment:
@hoyosjs to make sure I understand your question, you are asking about libcoreclrtraceptprovider.so and libdbgshim.so, correct?

libdbgshim.so doesn't need to ship with the runtime, so that is all good.
My understanding is that libcoreclrtraceptprovider.so is used for collecting LTTng traces on Linux. I am not sure if it is needed for dotnet-trace, or if it just needed for the old [perfcollect](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/linux-performance-tracing.md) script.

Reply:
Like Gregg libdbgshim.so doesn't need to ship with the runtime.  libcoreclrtraceptprovider.so does need to ship for the reasons Gregg stated.

Reply:
Thanks :)